integration-tests/test/rpc.spec.ts

```diff
@@ -293,6 +293,14 @@ describe('Basic RPC tests', () => {
   })
 
   describe('eth_estimateGas (returns the fee)', () => {
+    it('should return a gas estimate for txs with empty data', async () => {
+      const estimate = await l2Provider.estimateGas({
+        to: DEFAULT_TRANSACTION.to,
+        value: 0,
+      })
+      expect(estimate).to.be.eq(21000)
```

Comment:
Do transactions with empty data always return a gas estimate of 21000? Why? Can this be abused?

Reply:
This was implemented in https://github.com/ethereum-optimism/optimism/pull/695

Reply:
The idea was to never let the value be lower than `21000` because that breaks an expectation that Metamask had. With https://github.com/ethereum-optimism/optimism/pull/906 that line is removed